code/datums/components/toggle_attached_clothing.dm

```diff
@@ -82,6 +82,7 @@
 	unequip_deployable()
 	QDEL_NULL(deployable)
 	QDEL_NULL(toggle_action)
+	QDEL_NULL(undeployed_overlay)
```

Comment:
this is an MA right? shouldn't actually hold any references so there's no need to qdel it